src/base/Cellar.sol

```diff
@@ -1069,6 +1165,67 @@ contract Cellar is ERC4626, Ownable, Multicall {
 
     // ========================================= FEES LOGIC =========================================
 
+    /**
+     * @notice Emitted when High Watermark is reset.
+     */
+    event HighWatermarkReset();
+
+    /**
+     * @notice Resets High Watermark to equal current total assets.
+     * @notice This function can be abused by Strategists, so it should only be callable by governance.
+     */
+    function resetHighWatermark() external onlyOwner {
+        feeData.highWatermark = totalAssets();
+        emit HighWatermarkReset();
```

Comment:
Add a param for the new `totalAssets` value?

Reply:
Though this is set by governance, we'd want this to be accurate. If the reset value is above the current assets, the strategist has to make up that ground before earning fees. If the reset value is below, then the strategist earns performance fees for free.

Reply:
I mean for the event